doc/source/ray-air/examples/batch_tuning.ipynb

```diff
@@ -0,0 +1,1197 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6f5c2fbd-31d7-4262-94c8-1f6cea6f5938",
+   "metadata": {},
+   "source": [
+    "# Batch training & tuning on Ray Tune"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "db3ca5cc-6337-4569-b508-3784b3b6ab0c",
+   "metadata": {},
+   "source": [
+    "Batch training and tuning are common tasks in simple machine learning use-cases such as time series forecasting. They require fitting of simple models on multiple data batches corresponding to locations, products, etc.\n",
+    "\n",
+    "In the context of this notebook, batch training is understood as creating the same model(s) for different and separate data, or subsets of a dataset. This notebook showcases how to conduct batch training using [Ray Tune](https://docs.ray.io/en/latest/tune/index.html).\n",
+    "\n",
+    "![Batch training diagram](../../data/examples/images/batch-training.svg)\n",
+    "\n",
+    "For the data, we will use the [NYC Taxi dataset](https://www1.nyc.gov/site/tlc/about/tlc-trip-record-data.page). This popular tabular dataset contains historical taxi pickups by timestamp and location in NYC. To demonstrate batch training, we will simplify the data to a regression problem to predict `trip_duration` and use scikit-learn.\n",
+    "\n",
+    "To demonstrate how batch training can be parallelized, we will train a separate model for each dropoff location. This means we can use the `dropoff_location_id` column in the dataset to group the dataset into data batches. Then we will fit a separate model for each batch and evaluate it."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cdb02a92-a603-48b8-a23e-887ecaaf5470",
+   "metadata": {},
+   "source": [
+    "# Introduction to Ray Tune\n",
+    "\n",
+    "> **While Tune's main purpose is hyperparameter optimization, you can also use it as an execution engine to manage multiple independent compute workers.**  \n",
+    "\n",
+    "In this notebook, we will use [Ray Tune](https://docs.ray.io/en/latest/tune/index.html) to run separate, parallel training jobs, each as a different experiment, per dropoff location.  After all the Tune experiments have run, we will pick the best model per dropoff location.  \n",
+    "\n",
+    "Let us quickly walk through the [key concepts](https://docs.ray.io/en/latest/tune/key-concepts.html) you need to know to use Tune.\n",
+    "\n",
+    "- First, you define a `Search space` and pass that into a `Trainable` or `callable` function, that specifies the objective you want to tune.  The trainable function will be called for every experiment in the search space.\n",
```

Comment:
Can we use backticks for code/objects/classes/variables and italics (\*) for emphasis?